megamek/src/megamek/common/pathfinder/AbstractPathFinder.java

```diff
@@ -203,18 +207,24 @@ public boolean wasTimeoutEngaged() {
 
     private AdjacencyMap<E> adjacencyMap;
 
-    private PriorityQueue<E> candidates;
+    protected PriorityQueue<E> candidates;
```

Comment:
Why make this `protected`?

Reply:
At one point, I was inheriting from this class and needed to access the field. I guess that's not the case any more.